medical_appointment/views/medical_appointment_view.xml

```diff
@@ -6,222 +6,220 @@
+                        <div class="kanban_content">
+                            <div class="kanban_vignette semantic_html_override">
+                                <div class="kanban_details">
+                                    <h3>
+                                        <a type="open">
+                                            <field name="name" />
+                                        </a>
+                                    </h3>
+                                    <ul>
+                                        <li t-if="record.patient_id.raw_value">
+                                            <strong>
+                                                Patient:
+                                            </strong>
+                                            <field name="patient_id" />
+                                        </li>
+                                        <li t-if="record.physician_id.raw_value">
+                                            <strong>
+                                                Physician:
+                                            </strong>
+                                            <field name="physician_id" />
+                                        </li>
+                                        <li t-if="record.duration.raw_value">
+                                            <strong>Duration:</strong>
+                                            <field name="duration" />
+                                        </li>
+                                    </ul>
                                 </div>
                             </div>
-                        </t>
-                    </templates>
-                </kanban>
-            </field>
-        </record>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
 
-        <record model="ir.actions.act_window" id="medical_appointment_action">
-            <field name="name">Appointments</field>
-            <field name="type">ir.actions.act_window</field>
-            <field name="res_model">medical.appointment</field>
-            <field name="view_type">form</field>
-            <field name="view_mode">kanban,tree,form,calendar</field>
-            <field name="context">{}</field>
-            <!-- @TODO: Figure out this intent
-            field name="domain">['|',('patient_id.user_id.id','=',uid),('doctor.user_id.id','=',uid)]</field>
-            -->
-        </record>
+    <record model="ir.actions.act_window" id="medical_appointment_action">
+        <field name="name">Appointments</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">medical.appointment</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">kanban,tree,form,calendar</field>
+        <field name="context">{}</field>
+        <!-- @TODO: Figure out this intent
```

Comment:
I removed it completely, I don't think we need it in generic. It would be annoying in case there is a receptionist...